tests/components/homematicip_cloud/helper.py

```diff
@@ -84,17 +132,36 @@ class HomeTemplate(Home):
     _typeGroupMap = TYPE_GROUP_MAP
     _typeSecurityEventMap = TYPE_SECURITY_EVENT_MAP
 
-    def __init__(self, connection=None, home_name=""):
+    def __init__(self, connection=None, home_name="", test_devices=[], test_groups=[]):
         """Init template with connection."""
         super().__init__(connection=connection)
         self.label = "Access Point"
         self.name = home_name
         self.model_type = "HmIP-HAP"
         self.init_json_state = None
+        self.test_devices = test_devices
+        self.test_groups = test_groups
+
+    def _cleanup_json(self, json):
+        if self.test_devices is not None:
+            new_devices = {}
+            for json_device in json["devices"].items():
+                if json_device[1]["label"] in self.test_devices:
+                    new_devices.update([json_device])
+            json["devices"] = new_devices
+
+        if self.test_groups is not None:
+            new_groups = {}
+            for json_group in json["groups"].items():
+                if json_group[1]["label"] in self.test_groups:
+                    new_groups.update([json_group])
+            json["groups"] = new_groups
+
+        return json
 
     def init_home(self, json_path=HOME_JSON):
         """Init template with json."""
-        self.init_json_state = json.loads(load_fixture(HOME_JSON))
+        self.init_json_state = self._cleanup_json(json.loads(load_fixture(HOME_JSON)))
```

Comment:
So I don't think `json_path` is ever used in `init_home` and so it's always the same fixture. I think that it could help in loading the fixture once instead of every test. I would still do `json.loads` on each test in case the original data is mutated. 

```
# Put on top of file.
FIXTURE_DATA = load_fixture(HOME_JSON)
```

Reply:
Done